spanner/spanner_snippets/spanner: TestCustomerManagedEncryptionKeys failed

Note: #2111 was also for this test, but it was closed more than 10 days ago. So, I didn't mark it flaky.

----

commit: 1405686b18d746d2bd57330b3aed1dc955d971d9
buildURL: [Build Status](https://source.cloud.google.com/results/invocations/58caeb5e-b178-4b44-90b2-4fe5308178d9), [Sponge](http://sponge2/58caeb5e-b178-4b44-90b2-4fe5308178d9)
status: failed

In another test I saw this test producing panics. Will be skipping until this can be looked into shortly here. Here is the panic

<details>
  <summary>panic</summary>

```bash
panic: test timed out after 1h0m0s

goroutine 682 [running]:
testing.(*M).startAlarm.func1()
	/usr/local/go/src/testing/testing.go:1788 +0x8e
created by time.goFunc
	/usr/local/go/src/time/sleep.go:180 +0x31

goroutine 1 [chan receive, 59 minutes]:
testing.tRunner.func1()
	/usr/local/go/src/testing/testing.go:1225 +0x311
testing.tRunner(0xc000582340, 0xc00063fd18)
	/usr/local/go/src/testing/testing.go:1265 +0x13b
testing.runTests(0xc0000a0100, {0x14969a0, 0x6, 0x6}, {0x48ac2d, 0xd0ba0b, 0x14a88e0})
	/usr/local/go/src/testing/testing.go:1596 +0x43f
testing.(*M).Run(0xc0000a0100)
	/usr/local/go/src/testing/testing.go:1504 +0x51d
main.main()
	_testmain.go:53 +0x14b

goroutine 19 [select]:
go.opencensus.io/stats/view.(*worker).start(0xc0001f8380)
	/go/pkg/mod/go.opencensus.io@v0.23.0/stats/view/worker.go:276 +0xb9
created by go.opencensus.io/stats/view.init.0
	/go/pkg/mod/go.opencensus.io@v0.23.0/stats/view/worker.go:34 +0x92

goroutine 10 [select]:
github.com/googleapis/gax-go/v2.Sleep({0xeba5e8, 0xc000868660}, 0xc000868660)
	/go/pkg/mod/github.com/googleapis/gax-go/v2@v2.1.0/invoke.go:57 +0x85
cloud.google.com/go/longrunning.(*Operation).wait(0xc00067cb28, {0xeba5e8, 0xc000868660}, {0xeb5b58, 0xc000a7b7a0}, 0xeba620, 0xd47b98, {0x0, 0x0, 0x0})
	/go/pkg/mod/cloud.google.com/go@v0.94.0/longrunning/longrunning.go:156 +0xc2
cloud.google.com/go/longrunning.(*Operation).WaitWithInterval(0xc000637b58, {0xeba5e8, 0xc000868660}, {0xeb5b58, 0xc000a7b7a0}, 0xba8900, {0x0, 0xba8900, 0xc0008b4600})
	/go/pkg/mod/cloud.google.com/go@v0.94.0/longrunning/longrunning.go:142 +0x97
cloud.google.com/go/spanner/admin/database/apiv1.(*CreateBackupOperation).Wait(0xc00011b418, {0xeba5e8, 0xc000868660}, {0x0, 0x0, 0x0})
	/go/pkg/mod/cloud.google.com/go/spanner@v1.25.0/admin/database/apiv1/database_admin_client.go:1065 +0x90
github.com/GoogleCloudPlatform/golang-samples/spanner/spanner_snippets/spanner.createBackupWithCustomerManagedEncryptionKey({0xeba5e8, 0xc000868660}, {0xea2820, 0xc0000fc0c0}, {0xc000046150, 0x65}, {0xc000674c40, 0x1e}, {0xc0000465b0, 0x6f})
	/tmpfs/src/github/golang-samples/spanner/spanner_snippets/spanner/spanner_create_backup_with_encryption_key.go:66 +0x4ef
github.com/GoogleCloudPlatform/golang-samples/spanner/spanner_snippets/spanner.TestCustomerManagedEncryptionKeys(0xc000582ea0)
	/tmpfs/src/github/golang-samples/spanner/spanner_snippets/spanner/integration_test.go:462 +0x532
testing.tRunner(0xc000582ea0, 0xd47878)
	/usr/local/go/src/testing/testing.go:1259 +0x102
created by testing.(*T).Run
	/usr/local/go/src/testing/testing.go:1306 +0x35a

goroutine 35 [select, 5 minutes]:
google.golang.org/grpc.(*ccBalancerWrapper).watcher(0xc0005a0050)
	/go/pkg/mod/google.golang.org/grpc@v1.40.0/balancer_conn_wrappers.go:71 +0xa5
created by google.golang.org/grpc.newCCBalancerWrapper
	/go/pkg/mod/google.golang.org/grpc@v1.40.0/balancer_conn_wrappers.go:62 +0x246

goroutine 36 [chan receive, 5 minutes]:
google.golang.org/grpc.(*addrConn).resetTransport(0xc00064e840)
	/go/pkg/mod/google.golang.org/grpc@v1.40.0/clientconn.go:1219 +0x48f
created by google.golang.org/grpc.(*addrConn).connect
	/go/pkg/mod/google.golang.org/grpc@v1.40.0/clientconn.go:849 +0x147

goroutine 37 [select, 5 minutes]:
google.golang.org/grpc.(*ccBalancerWrapper).watcher(0xc0005a0190)
	/go/pkg/mod/google.golang.org/grpc@v1.40.0/balancer_conn_wrappers.go:71 +0xa5
created by google.golang.org/grpc.newCCBalancerWrapper
	/go/pkg/mod/google.golang.org/grpc@v1.40.0/balancer_conn_wrappers.go:62 +0x246

goroutine 38 [chan receive, 5 minutes]:
google.golang.org/grpc.(*addrConn).resetTransport(0xc00064eb00)
	/go/pkg/mod/google.golang.org/grpc@v1.40.0/clientconn.go:1219 +0x48f
created by google.golang.org/grpc.(*addrConn).connect
	/go/pkg/mod/google.golang.org/grpc@v1.40.0/clientconn.go:849 +0x147

goroutine 692 [select, 5 minutes]:
google.golang.org/grpc/internal/transport.(*controlBuffer).get(0xc0007ea5f0, 0x1)
	/go/pkg/mod/google.golang.org/grpc@v1.40.0/internal/transport/controlbuf.go:406 +0x11b
google.golang.org/grpc/internal/transport.(*loopyWriter).run(0xc000750780)
	/go/pkg/mod/google.golang.org/grpc@v1.40.0/internal/transport/controlbuf.go:533 +0x85
google.golang.org/grpc/internal/transport.newHTTP2Client.func3()
	/go/pkg/mod/google.golang.org/grpc@v1.40.0/internal/transport/http2_client.go:405 +0x65
created by google.golang.org/grpc/internal/transport.newHTTP2Client
	/go/pkg/mod/google.golang.org/grpc@v1.40.0/internal/transport/http2_client.go:403 +0x1f45

goroutine 315 [IO wait]:
internal/poll.runtime_pollWait(0x7ff1a03e8668, 0x72)
	/usr/local/go/src/runtime/netpoll.go:229 +0x89
internal/poll.(*pollDesc).wait(0xc0003e4800, 0xc00033a600, 0x0)
	/usr/local/go/src/internal/poll/fd_poll_runtime.go:84 +0x32
internal/poll.(*pollDesc).waitRead(...)
	/usr/local/go/src/internal/poll/fd_poll_runtime.go:89
internal/poll.(*FD).Read(0xc0003e4800, {0xc00033a600, 0x1020, 0x1020})
	/usr/local/go/src/internal/poll/fd_unix.go:167 +0x25a
net.(*netFD).Read(0xc0003e4800, {0xc00033a600, 0xc00033a605, 0x22})
	/usr/local/go/src/net/fd_posix.go:56 +0x29
net.(*conn).Read(0xc00006e5e0, {0xc00033a600, 0x3c, 0xc000724790})
	/usr/local/go/src/net/net.go:183 +0x45
crypto/tls.(*atLeastReader).Read(0xc00089e228, {0xc00033a600, 0x0, 0x40ba4d})
	/usr/local/go/src/crypto/tls/conn.go:777 +0x3d
bytes.(*Buffer).ReadFrom(0xc0004d1e78, {0xea2a20, 0xc00089e228})
	/usr/local/go/src/bytes/buffer.go:204 +0x98
crypto/tls.(*Conn).readFromUntil(0xc0004d1c00, {0xea8660, 0xc00006e5e0}, 0x1020)
	/usr/local/go/src/crypto/tls/conn.go:799 +0xe5
crypto/tls.(*Conn).readRecordOrCCS(0xc0004d1c00, 0x0)
	/usr/local/go/src/crypto/tls/conn.go:606 +0x112
crypto/tls.(*Conn).readRecord(...)
	/usr/local/go/src/crypto/tls/conn.go:574
crypto/tls.(*Conn).Read(0xc0004d1c00, {0xc0007dc000, 0x8000, 0x1})
	/usr/local/go/src/crypto/tls/conn.go:1277 +0x16f
bufio.(*Reader).Read(0xc0008db0e0, {0xc000638498, 0x9, 0x18})
	/usr/local/go/src/bufio/bufio.go:227 +0x1b4
io.ReadAtLeast({0xea27c0, 0xc0008db0e0}, {0xc000638498, 0x9, 0x9}, 0x9)
	/usr/local/go/src/io/io.go:328 +0x9a
io.ReadFull(...)
	/usr/local/go/src/io/io.go:347
golang.org/x/net/http2.readFrameHeader({0xc000638498, 0x9, 0x14a88e0}, {0xea27c0, 0xc0008db0e0})
	/go/pkg/mod/golang.org/x/net@v0.0.0-20210503060351-7fd8e65b6420/http2/frame.go:237 +0x6e
golang.org/x/net/http2.(*Framer).ReadFrame(0xc000638460)
	/go/pkg/mod/golang.org/x/net@v0.0.0-20210503060351-7fd8e65b6420/http2/frame.go:492 +0x95
google.golang.org/grpc/internal/transport.(*http2Client).reader(0xc00054c5a0)
	/go/pkg/mod/google.golang.org/grpc@v1.40.0/internal/transport/http2_client.go:1495 +0x41f
created by google.golang.org/grpc/internal/transport.newHTTP2Client
	/go/pkg/mod/google.golang.org/grpc@v1.40.0/internal/transport/http2_client.go:355 +0x18ef

goroutine 316 [select]:
google.golang.org/grpc/internal/transport.(*controlBuffer).get(0xc0006739f0, 0x1)
	/go/pkg/mod/google.golang.org/grpc@v1.40.0/internal/transport/controlbuf.go:406 +0x11b
google.golang.org/grpc/internal/transport.(*loopyWriter).run(0xc0008db1a0)
	/go/pkg/mod/google.golang.org/grpc@v1.40.0/internal/transport/controlbuf.go:533 +0x85
google.golang.org/grpc/internal/transport.newHTTP2Client.func3()
	/go/pkg/mod/google.golang.org/grpc@v1.40.0/internal/transport/http2_client.go:405 +0x65
created by google.golang.org/grpc/internal/transport.newHTTP2Client
	/go/pkg/mod/google.golang.org/grpc@v1.40.0/internal/transport/http2_client.go:403 +0x1f45

goroutine 309 [select, 59 minutes]:
google.golang.org/grpc.(*ccBalancerWrapper).watcher(0xc000673450)
	/go/pkg/mod/google.golang.org/grpc@v1.40.0/balancer_conn_wrappers.go:71 +0xa5
created by google.golang.org/grpc.newCCBalancerWrapper
	/go/pkg/mod/google.golang.org/grpc@v1.40.0/balancer_conn_wrappers.go:62 +0x246

goroutine 623 [select, 5 minutes]:
google.golang.org/grpc/internal/transport.(*controlBuffer).get(0xc000672410, 0x1)
	/go/pkg/mod/google.golang.org/grpc@v1.40.0/internal/transport/controlbuf.go:406 +0x11b
google.golang.org/grpc/internal/transport.(*loopyWriter).run(0xc000868540)
	/go/pkg/mod/google.golang.org/grpc@v1.40.0/internal/transport/controlbuf.go:533 +0x85
google.golang.org/grpc/internal/transport.newHTTP2Client.func3()
	/go/pkg/mod/google.golang.org/grpc@v1.40.0/internal/transport/http2_client.go:405 +0x65
created by google.golang.org/grpc/internal/transport.newHTTP2Client
	/go/pkg/mod/google.golang.org/grpc@v1.40.0/internal/transport/http2_client.go:403 +0x1f45

goroutine 310 [chan receive, 59 minutes]:
google.golang.org/grpc.(*addrConn).resetTransport(0xc000405b80)
	/go/pkg/mod/google.golang.org/grpc@v1.40.0/clientconn.go:1219 +0x48f
created by google.golang.org/grpc.(*addrConn).connect
	/go/pkg/mod/google.golang.org/grpc@v1.40.0/clientconn.go:849 +0x147

goroutine 111 [chan receive, 5 minutes]:
google.golang.org/grpc.(*addrConn).resetTransport(0xc0006cc000)
	/go/pkg/mod/google.golang.org/grpc@v1.40.0/clientconn.go:1219 +0x48f
created by google.golang.org/grpc.(*addrConn).connect
	/go/pkg/mod/google.golang.org/grpc@v1.40.0/clientconn.go:849 +0x147

goroutine 110 [select, 5 minutes]:
google.golang.org/grpc.(*ccBalancerWrapper).watcher(0xc000032190)
	/go/pkg/mod/google.golang.org/grpc@v1.40.0/balancer_conn_wrappers.go:71 +0xa5
created by google.golang.org/grpc.newCCBalancerWrapper
	/go/pkg/mod/google.golang.org/grpc@v1.40.0/balancer_conn_wrappers.go:62 +0x246

goroutine 622 [IO wait, 5 minutes]:
internal/poll.runtime_pollWait(0x7ff1a03e8da8, 0x72)
	/usr/local/go/src/runtime/netpoll.go:229 +0x89
internal/poll.(*pollDesc).wait(0xc0008d8980, 0xc000797000, 0x0)
	/usr/local/go/src/internal/poll/fd_poll_runtime.go:84 +0x32
internal/poll.(*pollDesc).waitRead(...)
	/usr/local/go/src/internal/poll/fd_poll_runtime.go:89
internal/poll.(*FD).Read(0xc0008d8980, {0xc000797000, 0xfd5, 0xfd5})
	/usr/local/go/src/internal/poll/fd_unix.go:167 +0x25a
net.(*netFD).Read(0xc0008d8980, {0xc000797000, 0xc000797043, 0x1a})
	/usr/local/go/src/net/fd_posix.go:56 +0x29
net.(*conn).Read(0xc000678000, {0xc000797000, 0x60, 0xc000783790})
	/usr/local/go/src/net/net.go:183 +0x45
crypto/tls.(*atLeastReader).Read(0xc00089e7b0, {0xc000797000, 0x0, 0x40ba4d})
	/usr/local/go/src/crypto/tls/conn.go:777 +0x3d
bytes.(*Buffer).ReadFrom(0xc0003a65f8, {0xea2a20, 0xc00089e7b0})
	/usr/local/go/src/bytes/buffer.go:204 +0x98
crypto/tls.(*Conn).readFromUntil(0xc0003a6380, {0xea8660, 0xc000678000}, 0xf97)
	/usr/local/go/src/crypto/tls/conn.go:799 +0xe5
crypto/tls.(*Conn).readRecordOrCCS(0xc0003a6380, 0x0)
	/usr/local/go/src/crypto/tls/conn.go:606 +0x112
crypto/tls.(*Conn).readRecord(...)
	/usr/local/go/src/crypto/tls/conn.go:574
crypto/tls.(*Conn).Read(0xc0003a6380, {0xc0005cc000, 0x8000, 0xc00003d3d8})
	/usr/local/go/src/crypto/tls/conn.go:1277 +0x16f
bufio.(*Reader).Read(0xc000868480, {0xc00087a1f8, 0x9, 0xc00005c800})
	/usr/local/go/src/bufio/bufio.go:227 +0x1b4
io.ReadAtLeast({0xea27c0, 0xc000868480}, {0xc00087a1f8, 0x9, 0x9}, 0x9)
	/usr/local/go/src/io/io.go:328 +0x9a
io.ReadFull(...)
	/usr/local/go/src/io/io.go:347
golang.org/x/net/http2.readFrameHeader({0xc00087a1f8, 0x9, 0xc0009ac7b0}, {0xea27c0, 0xc000868480})
	/go/pkg/mod/golang.org/x/net@v0.0.0-20210503060351-7fd8e65b6420/http2/frame.go:237 +0x6e
golang.org/x/net/http2.(*Framer).ReadFrame(0xc00087a1c0)
	/go/pkg/mod/golang.org/x/net@v0.0.0-20210503060351-7fd8e65b6420/http2/frame.go:492 +0x95
google.golang.org/grpc/internal/transport.(*http2Client).reader(0xc0001c90e0)
	/go/pkg/mod/google.golang.org/grpc@v1.40.0/internal/transport/http2_client.go:1495 +0x41f
created by google.golang.org/grpc/internal/transport.newHTTP2Client
	/go/pkg/mod/google.golang.org/grpc@v1.40.0/internal/transport/http2_client.go:355 +0x18ef

goroutine 691 [IO wait, 5 minutes]:
internal/poll.runtime_pollWait(0x7ff1a03e8f78, 0x72)
	/usr/local/go/src/runtime/netpoll.go:229 +0x89
internal/poll.(*pollDesc).wait(0xc00090a980, 0xc000190600, 0x0)
	/usr/local/go/src/internal/poll/fd_poll_runtime.go:84 +0x32
internal/poll.(*pollDesc).waitRead(...)
	/usr/local/go/src/internal/poll/fd_poll_runtime.go:89
internal/poll.(*FD).Read(0xc00090a980, {0xc000190600, 0x1021, 0x1021})
	/usr/local/go/src/internal/poll/fd_unix.go:167 +0x25a
net.(*netFD).Read(0xc00090a980, {0xc000190600, 0xc000190605, 0x1a})
	/usr/local/go/src/net/fd_posix.go:56 +0x29
net.(*conn).Read(0xc00011a0a0, {0xc000190600, 0x0, 0xc000729790})
	/usr/local/go/src/net/net.go:183 +0x45
crypto/tls.(*atLeastReader).Read(0xc00000e3d8, {0xc000190600, 0x0, 0x40ba4d})
	/usr/local/go/src/crypto/tls/conn.go:777 +0x3d
bytes.(*Buffer).ReadFrom(0xc0001ca5f8, {0xea2a20, 0xc00000e3d8})
	/usr/local/go/src/bytes/buffer.go:204 +0x98
crypto/tls.(*Conn).readFromUntil(0xc0001ca380, {0xea8660, 0xc00011a0a0}, 0x1021)
	/usr/local/go/src/crypto/tls/conn.go:799 +0xe5
crypto/tls.(*Conn).readRecordOrCCS(0xc0001ca380, 0x0)
	/usr/local/go/src/crypto/tls/conn.go:606 +0x112
crypto/tls.(*Conn).readRecord(...)
	/usr/local/go/src/crypto/tls/conn.go:574
crypto/tls.(*Conn).Read(0xc0001ca380, {0xc00056a000, 0x8000, 0xc00084e958})
	/usr/local/go/src/crypto/tls/conn.go:1277 +0x16f
bufio.(*Reader).Read(0xc0007506c0, {0xc0003b21f8, 0x9, 0xc00005c800})
	/usr/local/go/src/bufio/bufio.go:227 +0x1b4
io.ReadAtLeast({0xea27c0, 0xc0007506c0}, {0xc0003b21f8, 0x9, 0x9}, 0x9)
	/usr/local/go/src/io/io.go:328 +0x9a
io.ReadFull(...)
	/usr/local/go/src/io/io.go:347
golang.org/x/net/http2.readFrameHeader({0xc0003b21f8, 0x9, 0xc0007fe120}, {0xea27c0, 0xc0007506c0})
	/go/pkg/mod/golang.org/x/net@v0.0.0-20210503060351-7fd8e65b6420/http2/frame.go:237 +0x6e
golang.org/x/net/http2.(*Framer).ReadFrame(0xc0003b21c0)
	/go/pkg/mod/golang.org/x/net@v0.0.0-20210503060351-7fd8e65b6420/http2/frame.go:492 +0x95
google.golang.org/grpc/internal/transport.(*http2Client).reader(0xc00000cd20)
	/go/pkg/mod/google.golang.org/grpc@v1.40.0/internal/transport/http2_client.go:1495 +0x41f
created by google.golang.org/grpc/internal/transport.newHTTP2Client
	/go/pkg/mod/google.golang.org/grpc@v1.40.0/internal/transport/http2_client.go:355 +0x18ef

goroutine 641 [IO wait, 5 minutes]:
internal/poll.runtime_pollWait(0x7ff1a03e8bd8, 0x72)
	/usr/local/go/src/runtime/netpoll.go:229 +0x89
internal/poll.(*pollDesc).wait(0xc0001ba380, 0xc000838600, 0x0)
	/usr/local/go/src/internal/poll/fd_poll_runtime.go:84 +0x32
internal/poll.(*pollDesc).waitRead(...)
	/usr/local/go/src/internal/poll/fd_poll_runtime.go:89
internal/poll.(*FD).Read(0xc0001ba380, {0xc000838600, 0x1020, 0x1020})
	/usr/local/go/src/internal/poll/fd_unix.go:167 +0x25a
net.(*netFD).Read(0xc0001ba380, {0xc000838600, 0xc000838605, 0x1a})
	/usr/local/go/src/net/fd_posix.go:56 +0x29
net.(*conn).Read(0xc000010088, {0xc000838600, 0xd5a79147c6e00bf3, 0xc000784790})
	/usr/local/go/src/net/net.go:183 +0x45
crypto/tls.(*atLeastReader).Read(0xc00089e690, {0xc000838600, 0x0, 0x40ba4d})
	/usr/local/go/src/crypto/tls/conn.go:777 +0x3d
bytes.(*Buffer).ReadFrom(0xc00072a978, {0xea2a20, 0xc00089e690})
	/usr/local/go/src/bytes/buffer.go:204 +0x98
crypto/tls.(*Conn).readFromUntil(0xc00072a700, {0xea8660, 0xc000010088}, 0x1020)
	/usr/local/go/src/crypto/tls/conn.go:799 +0xe5
crypto/tls.(*Conn).readRecordOrCCS(0xc00072a700, 0x0)
	/usr/local/go/src/crypto/tls/conn.go:606 +0x112
crypto/tls.(*Conn).readRecord(...)
	/usr/local/go/src/crypto/tls/conn.go:574
crypto/tls.(*Conn).Read(0xc00072a700, {0xc00023c000, 0x8000, 0xc000656c58})
	/usr/local/go/src/crypto/tls/conn.go:1277 +0x16f
bufio.(*Reader).Read(0xc0008c2960, {0xc000262498, 0x9, 0xc00005d400})
	/usr/local/go/src/bufio/bufio.go:227 +0x1b4
io.ReadAtLeast({0xea27c0, 0xc0008c2960}, {0xc000262498, 0x9, 0x9}, 0x9)
	/usr/local/go/src/io/io.go:328 +0x9a
io.ReadFull(...)
	/usr/local/go/src/io/io.go:347
golang.org/x/net/http2.readFrameHeader({0xc000262498, 0x9, 0xc0009ac300}, {0xea27c0, 0xc0008c2960})
	/go/pkg/mod/golang.org/x/net@v0.0.0-20210503060351-7fd8e65b6420/http2/frame.go:237 +0x6e
golang.org/x/net/http2.(*Framer).ReadFrame(0xc000262460)
	/go/pkg/mod/golang.org/x/net@v0.0.0-20210503060351-7fd8e65b6420/http2/frame.go:492 +0x95
google.golang.org/grpc/internal/transport.(*http2Client).reader(0xc0007ce1e0)
	/go/pkg/mod/google.golang.org/grpc@v1.40.0/internal/transport/http2_client.go:1495 +0x41f
created by google.golang.org/grpc/internal/transport.newHTTP2Client
	/go/pkg/mod/google.golang.org/grpc@v1.40.0/internal/transport/http2_client.go:355 +0x18ef

goroutine 658 [select, 5 minutes]:
google.golang.org/grpc/internal/transport.(*controlBuffer).get(0xc00011ef00, 0x1)
	/go/pkg/mod/google.golang.org/grpc@v1.40.0/internal/transport/controlbuf.go:406 +0x11b
google.golang.org/grpc/internal/transport.(*loopyWriter).run(0xc0008c2a20)
	/go/pkg/mod/google.golang.org/grpc@v1.40.0/internal/transport/controlbuf.go:533 +0x85
google.golang.org/grpc/internal/transport.newHTTP2Client.func3()
	/go/pkg/mod/google.golang.org/grpc@v1.40.0/internal/transport/http2_client.go:405 +0x65
created by google.golang.org/grpc/internal/transport.newHTTP2Client
	/go/pkg/mod/google.golang.org/grpc@v1.40.0/internal/transport/http2_client.go:403 +0x1f45
FAIL	github.com/GoogleCloudPlatform/golang-samples/spanner/spanner_snippets/spanner	3600.022s
```
</details>

--- a/spanner/spanner_snippets/spanner/integration_test.go
+++ b/spanner/spanner_snippets/spanner/integration_test.go
@@ -428,13 +428,12 @@ func TestCreateDatabaseWithRetentionPeriodSample(t *testing.T) {
 }
 
 func TestCustomerManagedEncryptionKeys(t *testing.T) {
-	t.Skip("https://github.com/GoogleCloudPlatform/golang-samples/issues/2230")
 	if os.Getenv("GOLANG_SAMPLES_E2E_TEST") == "" {
 		t.Skip("GOLANG_SAMPLES_E2E_TEST not set")
 	}
 	tc := testutil.SystemTest(t)
 	t.Parallel()
-
+	startTime := time.Now()
 	instName, dbName, cleanup := initTest(t, randomID())
 	defer cleanup()
 
@@ -456,7 +455,7 @@ func TestCustomerManagedEncryptionKeys(t *testing.T) {
 		keyId,
 	)
 
-	ctx, cancel := context.WithTimeout(context.Background(), time.Hour)
+	ctx, cancel := context.WithTimeout(context.Background(), 90*time.Minute)
 	defer cancel()
 
 	// Create an encrypted database. The database is automatically deleted by the cleanup function.
@@ -465,6 +464,7 @@ func TestCustomerManagedEncryptionKeys(t *testing.T) {
 	}
 	out := b.String()
 	assertContains(t, out, fmt.Sprintf("Created database [%s] using encryption key %q", dbName, kmsKeyName))
+	t.Logf("create database operation took: %v\n", time.Since(startTime))
 
 	// Try to create a backup of the encrypted database and delete it after the test.
 	backupId := fmt.Sprintf("enc-backup-%s", randomID())
@@ -475,6 +475,7 @@ func TestCustomerManagedEncryptionKeys(t *testing.T) {
 	out = b.String()
 	assertContains(t, out, fmt.Sprintf("backups/%s", backupId))
 	assertContains(t, out, fmt.Sprintf("using encryption key %s", kmsKeyName))
+	t.Logf("create backup operation took: %v\n", time.Since(startTime))
 
 	// Try to restore the encrypted database and delete the restored database after the test.
 	restoredName := fmt.Sprintf("%s/databases/rest-enc-%s", instName, randomID())
@@ -484,6 +485,7 @@ func TestCustomerManagedEncryptionKeys(t *testing.T) {
 	out = runBackupSampleWithRetry(ctx, t, restoreFunc, restoredName, backupId, "failed to restore database with customer managed encryption key", 10)
 	assertContains(t, out, fmt.Sprintf("Database %s restored", dbName))
 	assertContains(t, out, fmt.Sprintf("using encryption key %s", kmsKeyName))
+	t.Logf("restore backup operation took: %v\n", time.Since(startTime))
 }
 
 func TestCreateDatabaseWithDefaultLeaderSample(t *testing.T) {

